components/sts/org.wso2.carbon.identity.sts.mgt.ui/src/main/resources/web/generic-sts/sts.jsp

```diff
@@ -186,9 +187,9 @@
                 </td>
                 <td>
                     <div style="height:30px;">
-                        <a href="javascript:document.location.href='<%=serverUrl+"?wsdl"%>'"
+                        <a href="javascript:document.location.href='<%=Encode.forHtmlAttribute(serverUrl)+"?wsdl"%>'"
```

Comment:
Here javascript execution context is being invoked. Thus javascript encoding should be performed. However, serverurl seems to be constructed internally and trusted. So it's not necessary to apply encoding here